server/app/assets/stylesheets/northstar/_uswds-theme-custom-styles.scss

```diff
@@ -151,6 +182,11 @@ See the radio-option below as an example.
   .grid-container {
     grid-template-columns: 1fr;
   }
+
+  // Put some vertical space between two buttons
+  .usa-button + .usa-button {
```

Comment:
I'm a little suspicious of this, since buttons placed next to each other in the dom might be stacked vertically (And thus the margin makes sense) but could also be stacked horizontally, right? In which case it's not clear we'd want the margin.

Looking a bit more, it feels like the button group component might handle this? https://designsystem.digital.gov/components/button-group/